smart_ats/jobs/models.py

```diff
@@ -50,3 +53,36 @@ def archive(self):
 
     def __str__(self):
         return f'{self.title} {self.company.name} {self.author.first_name}'
+
+class JobApplication(TimeStampedModel):
+
+    STATUS = Choices(
+        ('DRAFT', _('Draft')),
+        ('ACTIVE', _('Active')),
+        ('SHORTLISTED',_('Short-listed')),
+        ('REJECTED',_('Rejected')),
+        ('ARCHIVED', _('Archived')),
+    )
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    job = models.ForeignKey(Job, on_delete=models.CASCADE)
+    state = FSMField(default=STATUS.DRAFT, choices=STATUS)
+    data = JSONField()
+    cv_url = models.URLField(max_length=100)
+
+    @transition(field=state, source=STATUS.DRAFT, target=STATUS.ACTIVE)
+    def activate(self):
+        pass
+
+    @transition(field=state, source=STATUS.ACTIVE, target=['Short-listed','Rejected','Archived'])
```

Comment:
please roll back to 0001 migration and recreate the migration to adapt for new changes.
please use pattern of STATUS.SHORTLISTED, etc